packages/calypso-e2e/src/types/env-variables.types.ts

```diff
@@ -8,6 +8,14 @@ export type EnvVariables = {
 export type ViewportName = string;
 export type TestLocales = string[];
 export type JetpackTarget = 'remote-site' | 'wpcom-production' | 'wpcom-deployment';
+export type AtomicVariation =
+	| 'default'
+	| 'php-old'
+	| 'php-new'
+	| 'wp-beta'
+	| 'wp-previous'
+	| 'private'
+	| 'ecomm-plan';
```

Comment:
**question:non-blocking** - if I'm understanding correctly, we want to eventually run the set of E2E against all these environments?

Reply:
Yep! 🚀 

When this PR merges, we'll actually already be doing it for the Jetpack Atomic tests! 😁 But we could extend this to our Gutenberg tests in the future too, which could be pretty rad.